db/mongoose.js

```diff
@@ -234,8 +234,9 @@ const getIndex = async (req, res, next) => {
       theResults.push(index[i]);
     // }
   }
-    console.log("theResults", theResults)
-    res.json(theResults);
+
+  console.log("theResults", theResults.slice(-30))
+  res.json(theResults);
```

Comment:
slicer is only slicing the console log result, and what we need isnt this but we need to make a new endpoint as mentioned `/ustonks/index-history-daily` getting only daily results and not the results of each 5 minutes